server/apps/distributed-app/README.adoc

```diff
@@ -19,7 +19,7 @@ Third party compulsory dependencies:
 ----
 $ docker run -d --network james -p 9042:9042 --name=cassandra cassandra:3.11.10
 $ docker run -d --network james -p 9200:9200 --name=opensearch --env 'discovery.type=single-node' opensearchproject/opensearch:2.1.0
-$ docker run -d --network james -p 5672:5672 -p 15672:15672 --name=rabbitmq rabbitmq:3.9.18-management
+$ docker run -d --network james -p 5672:5672 -p 15672:15672 --name=rabbitmq rabbitmq:3.12.1-management
```

Comment:
Idem same as 3.7.x: running with 3.9 should still be possible for users who wants.